lib/jekyll/configuration.rb

```diff
@@ -239,13 +244,13 @@ def backwards_compatibilize
         config[option].map!(&:to_s)
       end
 
-      if (config['kramdown'] || {}).key?('use_coderay')
+      if (config["kramdown"] || {}).key?("use_coderay")
         Jekyll::Deprecator.deprecation_message "Please change 'use_coderay'" \
           " to 'enable_coderay' in your configuration file."
-        config['kramdown']['use_coderay'] = config['kramdown'].delete('enable_coderay')
+        config["kramdown"]["use_coderay"] = config["kramdown"].delete("enable_coderay")
       end
 
-      if config.fetch('markdown', 'kramdown').to_s.downcase.eql?("maruku")
+      if config.fetch("markdown", "kramdown").to_s.casecmp("maruku")
```

Comment:
This won't work the way you think it works and IMO `String#casecmp` is broken.  It'll match "mark", it's also non-boolean.  This will cause us trouble in the long run,.  However, I will concede that it will work if you correct it to `config.fetch("markdown", "kramdown").to_s.casecmp("maruku") == 0` which will match any case but only the whole value. instead of just part of the value.